packages/styled-components/src/native/index.js

```diff
@@ -1,7 +1,207 @@
 // @flow
 
 /* eslint-disable import/no-unresolved */
-import * as reactNative from 'react-native';
+import {
+  AccessibilityInfo,
+  ActivityIndicator,
+  ReactNativeART,
```

Comment:
ART is gone from RNW@0.11 and several other modules are already deprecated in RN or soon will be